gossip/src/crds_gossip.rs

```diff
@@ -306,13 +306,13 @@ impl CrdsGossip {
         self_pubkey: &Pubkey,
         thread_pool: &ThreadPool,
         now: u64,
-        timeouts: &HashMap<Pubkey, u64>,
+        timeouts: &CrdsTimeouts,
     ) -> usize {
         let mut rv = 0;
         if now > self.pull.crds_timeout {
             //sanity check
             assert_eq!(timeouts[self_pubkey], std::u64::MAX);
-            assert!(timeouts.contains_key(&Pubkey::default()));
+            assert_ne!(timeouts[&Pubkey::default()], 0u64);
```

Comment:
Idk if this assert is adding any value anymore since keys aren't really inserted into timeouts struct anymore. Maybe assert that this timeout is equal to `timeouts.default_timeout`?

Reply:
changed to `_eq(..., self.pull.crds_timeout)`;

Reply:
Reverted this back to `_ne!(..., 0)` since there are some tests that stake default pubkey, and they fails.
The asserts are not `debug_assert` since they don't matter much anymore.

Reply:
`s/not/now/`, right?